.github/workflows/cache_server.test.yml

```diff
@@ -0,0 +1,49 @@
+name: Cache-Server Tester
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events
+  push:
+  pull_request:
+
+  workflow_dispatch:
+
+jobs:
+
+  deployment:
+    services:
+      postgres:
+       image: postgres
+       env:
+        DB_USERNAME: ${{secrets.DB_USERNAME}}
+        DB_PASSWORD: ${{secrets.DB_PASSWORD}}
+        DB_HOST: 'postgres'  
+       ports:
+       - 5432:5432
+      
+    runs-on: ubuntu-latest
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        run: npm install
+        
+      - name: Docker Build
```

Comment:
i think we shouldn't use docker as part of this workflow because it will increase the time it takes for this workflow to build. let the self hosted containers by the actions handle that.